### PR TITLE
Inductor cpp wrapper: support QConv

### PR DESCRIPTION
**Pull request by chunyuan-w**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #112379
* #112378
* __->__ #112373

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler

### CODE REVIEW

**Review by jgong5 — APPROVED** [review]

File: torch/_inductor/ir.py [hunk 1]
```diff
@@ -5529,7 +5561,40 @@ def __init__(
...
+            layout,
+            inputs,
+            constant_args,
+            None,
+            kernel="torch.ops.onednn.qconv2d_pointwise.binary",
+            cpp_kernel="onednn::qconv2d_pointwise",
+        )
+        self.cpp_kernel_overlad_name = "binary"
+        self.cpp_kernel_key = "qconv2d_pointwise_binary"
+        self.cpp_op_schema = """
```
Comment (desertfire) [inline torch/_inductor/ir.py:5574]: Is it possible to read this kind of schema from somewhere instead of keep adding them for more and more ops?

**Review by desertfire — APPROVED** [review]

File: torch/_inductor/ir.py (hunk 1 above)
Comment (chunyuan-w) [reply torch/_inductor/ir.py:5574]: Let me submit a subsequent PR to clean this up. Created an issue to track it: https://github.com/pytorch/pytorch/issues/112552

**Pull request closed without merging**